src/components/posts/State.tsx

```diff
@@ -0,0 +1,15 @@
+import { atom, atomFamily } from 'recoil'
+import { IComment } from './comments/CommentList'
+import { IPost } from './PostList'
+
+const postAtom = atom<IPost[]>({
+  key: 'post',
+  default: [],
+})
+
+export default postAtom
```

Comment:
There are multiple atoms being exported in this file, so it would be more consistent if this was not a default export.